runcards.py

```diff
@@ -33,86 +33,122 @@ def call_makeDataCard(cmd):
         cmd_crA += "--input=config/SUEP_inputs_{era}.yaml --era={era}"
         cmd_crA = cmd_crA.format(signal=n, era=year)
 
         cmd_crB = "python3 makeDataCard.py --channel cat_crB "
-        cmd_crB += "--variable B_SUEP_nconst_Cluster "
-        cmd_crB += "--stack {signal} data "
-        #cmd_crB += "--bins " + new_bins + " "
-        #cmd_crB += "--rebin 25 " 
+        cmd_crB += "--variable B_SUEP_nconst_Cluster70 "
+        cmd_crB += "--stack {signal} expected data "
+        cmd_crB += "--bins 0 500 "
+        #cmd_crB += "--rebin 300 " 
         cmd_crB += "--input=config/SUEP_inputs_{era}.yaml --era={era}"
         cmd_crB = cmd_crB.format(signal=n, era=year)
 
         cmd_crC = "python3 makeDataCard.py --channel cat_crC "
-        cmd_crC += "--variable C_SUEP_nconst_Cluster "
-        cmd_crC += "--stack {signal} data "
-        #cmd_crC += "--bins " + new_bins + " "
-        #cmd_crC += "--rebin 25 " 
+        cmd_crC += "--variable C_SUEP_nconst_Cluster70 "
+        cmd_crC += "--stack {signal} expected data "
+        cmd_crC += "--bins 0 500 "
+        #cmd_crC += "--rebin 300 " 
         cmd_crC += "--input=config/SUEP_inputs_{era}.yaml --era={era}"
         cmd_crC = cmd_crC.format(signal=n, era=year)
 
         cmd_crD = "python3 makeDataCard.py --channel cat_crD "
-        cmd_crD += "--variable D_SUEP_nconst_Cluster "
-        cmd_crD += "--stack {signal} data "
-        #cmd_crD += "--bins " + new_bins + " "
-        #cmd_crD += "--rebin 25 " 
+        cmd_crD += "--variable D_SUEP_nconst_Cluster70 "
+        cmd_crD += "--stack {signal} expected data "
+        cmd_crD += "--bins 0 500 "
+        #cmd_crD += "--rebin 300 " 
         cmd_crD += "--input=config/SUEP_inputs_{era}.yaml --era={era}"
         cmd_crD = cmd_crD.format(signal=n, era=year)
 
         cmd_crE = "python3 makeDataCard.py --channel cat_crE "
-        cmd_crE += "--variable E_SUEP_nconst_Cluster "
-        cmd_crE += "--stack {signal} data "
-        #cmd_crE += "--bins " + new_bins + " "
-        #cmd_crE += "--rebin 25 " 
+        cmd_crE += "--variable E_SUEP_nconst_Cluster70 "
+        cmd_crE += "--stack {signal} expected data "
+        cmd_crE += "--bins 0 500 "
+        #cmd_crE += "--rebin 300 " 
         cmd_crE += "--input=config/SUEP_inputs_{era}.yaml --era={era}"
         cmd_crE = cmd_crE.format(signal=n, era=year)
 
-        cmd_crF = "python3 makeDataCard.py --channel cat_crF "
-        cmd_crF += "--variable F_SUEP_nconst_Cluster "
-        cmd_crF += "--stack {signal} data "
-        #cmd_crF += "--bins " + new_bins + " "
-        #cmd_crF += "--rebin 25 " 
-        cmd_crF += "--input=config/SUEP_inputs_{era}.yaml --era={era}"
-        cmd_crF = cmd_crF.format(signal=n, era=year)
+        cmd_crF1 = "python3 makeDataCard.py --channel Bin1crF "
+        cmd_crF1 += "--variable F_SUEP_nconst_Cluster70 "
+        cmd_crF1 += "--stack {signal} expected data "
+        cmd_crF1 += "--bins 70 100 "
```

Comment:
Won't this rebin as [0,70,100,inf]? Don't we only want [70,100]? Is that being selected elsewhere?

Reply:
This might be a dumb comment, rebin_piecewise should rebin as you say it.